cmd/corectl/main.go

```diff
@@ -200,25 +200,39 @@ func createToken(client *rpc.Client, args []string) {
 	}
 	flags.Parse(args)
 	args = flags.Args()
-	if len(args) < 1 {
+	if len(args) == 2 && *flagNet || len(args) < 1 || len(args) > 2 {
 		fatalln(usage)
 	}
 
-	req := struct {
-		ID, Type string
-	}{
-		ID:   args[0],
-		Type: map[bool]string{true: "network", false: "client"}[*flagNet],
-	}
+	req := struct{ ID string }{args[0]}
 	var tok accesstoken.Token
 	err := client.Call(context.Background(), "/create-access-token", req, &tok)
 	if err != nil {
 		fatalln("rpc error:", err)
 	}
 	fmt.Println(tok.Token)
 
-	if *flagNet {
+	req := struct {
+		Guard_Type string
```

Comment:
nit: I'd rather we use json tags here, instead of underscores in field names. (At least I assume that's what the underscores are for?)